primitives/src/crh/poseidon/batched_crh.rs

```diff
@@ -0,0 +1,537 @@
+extern crate rand;
```

Comment:
Let's add a comment that we implement batch evaluation for x^{-1}-Poseidon.

Reply:
Done